packages/report-e2e-tests/jest.config.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const baseConfig = require('../../src/tests/unit/jest.config');
```

Comment:
Will we eventually have a separate jest.base.config, like tsconfig.base.json?

Reply:
Yes! I actually tried to do that in this PR, but it required changing almost a dozen more files, so I noted it as out of scope and have a separate branch ready to go with that :)